config/initializers/git_info.rb

```diff
@@ -0,0 +1,7 @@
+module Git
+    `which git`
+    if $?.success?
+      BRANCH = `git symbolic-ref HEAD 2>/dev/null | cut -d"/" -f 3`.strip
+      COMMIT = `git log --pretty=format:'%h' -n 1`.strip
+    end
```

Comment:
@jordimassaguerpla I think that we could do that if picking it from git is not working. My reasons:

1. I'd prefer to stick to git, since it would be more reliable than the `VERSION` file.
2. There's one case in which the git command won't work: when using a distro package (e.g. the RPM that we produce for openSUSE; good catch @jordimassaguerpla :P).

So, pick it from git if you can, but if you can't, just stick to the `VERSION` file.